assets/common.sh

```diff
@@ -41,14 +41,23 @@ setup_kubectl() {
     # Optional. The address and port of the API server. Requires token.
     local server
     server="$(jq -r '.source.server // ""' < "$payload")"
-    # Optional. A certificate file for the certificate authority.
+    # Optional. A file to read the certificate from.
+    local certificate_authority_file
+    certificate_authority_file="$(jq -r '.source.certificate_authority_file // ""' < "$payload")"
+    # Optional. A certificate for the certificate authority.
     local certificate_authority
-    certificate_authority="$(jq -r '.source.certificate_authority // ""' < "$payload")"
+    certificate_authority="$(jq -r '.source.certificate_authority // "${CERTIFICATE_AUTHORITY_FILE_CONTENTS}"' < "$payload")"
```

Comment:
Sorry, I'm not sure when is CERTIFICATE_AUTHORITY_FILE_CONTENTS initialized. I guess L56 exports this variable, but I can't imagine why is this script called twice or more.

@superbrothers please fix my understanding if I made a mistake.